sockeye/arguments.py

```diff
@@ -640,6 +707,34 @@ def add_vocab_args(params):
                         help='Pad vocabulary to a multiple of this integer. Default: %(default)s.')
 
 
+def add_mono_vocab_args(params):
+    params.add_argument('--mono-vocab',
+                        required=False,
+                        default=None,
+                        help='Existing vocabulary (JSON).')
+    params.add_argument('--mono-factor-vocabs',
```

Comment:
What about we merge this with `--mono-factors-use-mono-vocab`? If a path is specified, use it; if "default" (or something to this effect), then the data`s vocab is used.